binding.gyp

```diff
@@ -1,10 +1,16 @@
 {
+  "variables": { 
+    "target_arch": "<!(uname -p)"
```

Comment:
You shouldn't need to set `target_arch`, it's supposed to be an implicit variable.